### ISSUE TITLE
add support for OAI-PMH 

### ISSUE THREAD

**Issue by peterstadler** [Edirom/WeGA-WebApp#259]
some pointers:
* https://www.openarchives.org/pmh/
* exist-open thread ["OAI_PMH packages?"](https://exist-open.markmail.org/thread/da55ei3usiyqo2z2)

**Comment by peterstadler** [comment]
BASE has "[Goldene Regeln für Quellenbetreiber](https://www.base-search.net/about/de/faq_oai.php)" which provide good guidelines.

In general, I believe we'd need to implement a dedicated OAI endpoint which is capable to respond to the 6 request verbs as defined by the [standard](https://www.openarchives.org/OAI/openarchivesprotocol.html#ProtocolMessages).
For "GetRecord", this could look like e.g. `https://weber-gesamtausgabe.de/oai?verb=GetRecord&identifier=A040521&metadataPrefix=oai_dc`

**Comment by riedde** [comment]
Yes, we should extend the basics I tried to develop in the nearer future
